chartfx-math/src/main/java/de/gsi/math/MathGenBase.java

```diff
@@ -3758,70 +2839,20 @@ public static double NormQuantile(double p) {
         return quantile;
     }
 
-    public static double PeakToPeak(double[] data) {
-        return PeakToPeak(data, data.length);
-    }
-
-    /**
-     * @param data the input vector
-     * @param length &lt;= data.length elements to be used
-     * @return peak-to-peak value of vector element
-     */
-    public static double PeakToPeak(double[] data, int length) {
-        return Math.abs(Maximum(data, length) - Minimum(data, length));
-    }
-
-    public static float PeakToPeak(float[] data) {
-        return PeakToPeak(data, data.length);
-    }
-
-    /**
-     * @param data the input vector
-     * @param length &lt;= data.length elements to be used
-     * @return peak-to-peak value of vector element
-     */
-    public static float PeakToPeak(float[] data, int length) {
-        return Math.abs(Maximum(data, length) - Minimum(data, length));
-    }
-
-    public static int PeakToPeak(int[] data) {
-        return PeakToPeak(data, data.length);
-    }
-
-    /**
-     * @param data the input vector
-     * @param length &lt;= data.length elements to be used
-     * @return peak-to-peak value of vector element
-     */
-    public static int PeakToPeak(int[] data, int length) {
-        return Math.abs(Maximum(data, length) - Minimum(data, length));
-    }
-
-    public static long PeakToPeak(long[] data) {
-        return PeakToPeak(data, data.length);
-    }
-
-    /**
-     * @param data the input vector
-     * @param length &lt;= data.length elements to be used
-     * @return peak-to-peak value of vector element
-     */
-    public static long PeakToPeak(long[] data, int length) {
-        return Math.abs(Maximum(data, length) - Minimum(data, length));
-    }
-
-    public static short PeakToPeak(short[] data) {
-        return PeakToPeak(data, data.length);
+    //// codegen: double -> float, int, long, short
+    public static double peakToPeak(double[] data) {
+        return peakToPeak(data, data.length);
     }
 
     /**
      * @param data the input vector
      * @param length &lt;= data.length elements to be used
      * @return peak-to-peak value of vector element
      */
-    public static short PeakToPeak(short[] data, int length) {
-        return (short) Math.abs(Maximum(data, length) - Minimum(data, length));
+    public static double peakToPeak(double[] data, int length) {
+        return java.lang.Math.abs(maximum(data, length) - minimum(data, length)); //// codegen: returncast short
```

Comment:
Codacy found an issue: [Unnecessary use of fully qualified name 'java.lang.Math.abs' due to existing implicit import 'java.lang.*'](https://app.codacy.com/manual/GSI/chart-fx/pullRequest?prid=5780327)